contracts/SpokePool.sol

```diff
@@ -1330,10 +1343,6 @@ abstract contract SpokePool is
         // Verify depositor is a valid EVM address.
         params.depositor.checkAddress();
 
-        // Check that deposit route is enabled for the input token. There are no checks required for the output token
-        // which is pulled from the relayer at fill time and passed through this contract atomically to the recipient.
-        if (!enabledDepositRoutes[params.inputToken.toAddress()][params.destinationChainId]) revert DisabledRoute();
```

Comment:
third open question: removing this check now allows depositor to set any value for `destinationChainId` they want to. This opens up griefing vectors against offchain components that we need to make sure relayers and dataworkers protect against

Reply:
What griefing vectors? Just that people can send completely invalid deposits?

Reply:
yeah, basically you can set the destination chain ID so high that validators can't handle it. Shouldn't be an issue if they use BN.js